package.json

```diff
@@ -7,7 +7,12 @@
   "devDependencies": {
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.13.8",
     "@craco/craco": "^5.7.0",
-    "@ethersproject/experimental": "^5.2.0",
+    "@ethersproject/experimental": "^5.4.0",
+    "@gnosis.pm/safe-apps-web3-react": "^0.6.0",
+    "@graphql-codegen/cli": "1.21.5",
```

Comment:
lots of stuff we dont need...

Reply:
Shouldn't then we just not add them as dependencies?

Reply:
but a lot of files depend on these packages existing in compile time so they need to remain lest we remove the files importing them